.prettierrc.js

```diff
@@ -0,0 +1,20 @@
+module.exports = {
+  semi: false,
+  singleQuote: true,
+  trailingComma: 'none',
+  overrides: [
+    {
+      files: '*.md',
+      options: {
+        embeddedLanguageFormatting: 'off',
+        singleQuote: false
+      }
+    },
+    {
+      files: '*.scss',
+      options: {
+        singleQuote: false
+      }
+    }
```

Comment:
If we're only running prettier against `json`, `md`, `yaml` and `yml` files will this override ever be applied?

Reply:
For context, this is for compatibility

Text editors and IDEs also pick up the Prettier config

Should anyone want to turn on auto formatting (by choice) this stops Sass quotes being flipped to "single"